lib/routes/smh/author.ts

```diff
@@ -0,0 +1,66 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+
+export const route: Route = {
+    path: '/by/:authorSlug',
+    categories: ['programming'],
+    example: '/smh/by/natassia-chrysanthos-h17jwj',
+    parameters: { authorSlug: 'Slug of the author profile URL' },
+    name: 'Articles by an author',
+    maintainers: ['yue-dongchen'],
+    handler: async (ctx) => {
+        const baseUrl = 'https://smh.com.au';
+        const { authorSlug = 'natassia-chrysanthos-h17jwj' } = ctx.req.param();
+        const link = `https://smh.com.au/by/${authorSlug}`;
+
+        const response = await ofetch(link);
```

Comment:
Please use the GraphQL endpoint at `api.smh.com.au/graphql` instead.

Reply:
Thank you. I will do it in coming weeks.